.changes/nextrelease/use-phpunit-framework-test

```diff
@@ -0,0 +1,7 @@
+[
+   {
+       "type": "enhancement",
+       "category": "Test",
+       "description": "use PHPUnit\Framework\TestCase instead of PHPUnit_Framework_TestCase"
```

Comment:
Some of these characters will be removed due to backslashing. Can you also please capitalize the first letter in the description contents?

Reply:
Done